src/components/sequencing/SequenceEditor.svelte

```diff
@@ -262,17 +262,25 @@
     }
   }
 
-  function downloadOutputFormat() {
+  function downloadOutputFormat(outputFormat: IOutputFormat) {
```

Comment:
We just added `downloadBlob` and `downloadJSON` to `utilities/generic.ts`. Do you mind utilizing those functions here?

Reply:
Thanks!